caringcaribou/modules/dump.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import print_function
-from lib.can_actions import CanActions
-from lib.common import msg_to_candump_format, parse_int_dec_or_hex
+from caringcaribou.utils.can_actions import CanActions
+from caringcaribou.utils.common import msg_to_candump_format, parse_int_dec_or_hex
 from modules.send import FILE_LINE_COMMENT_PREFIX
```

Comment:
This probably needs to import `caringcaribou.modules.send`, right? When I run the dump module as-is from this PR I get the following error:

```
$ cc.py dump

----------------------
CARING CARIBOU v0.4dev
----------------------

Loading module 'dump'

Traceback (most recent call last):
  File "/usr/local/bin/cc.py", line 11, in <module>
    load_entry_point('caringcaribou==0.4.dev0', 'console_scripts', 'cc.py')()
  File "/usr/local/lib/python3.8/dist-packages/caringcaribou-0.4.dev0-py3.8.egg/caringcaribou/caringcaribou.py", line 115, in main
    cc_mod = load_module(args.module).load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2445, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2451, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.8/dist-packages/caringcaribou-0.4.dev0-py3.8.egg/caringcaribou/modules/dump.py", line 4, in <module>
    from modules.send import FILE_LINE_COMMENT_PREFIX
ModuleNotFoundError: No module named 'modules'
```

Reply:
Missed the modules import, tried to fix it in the last push